src/iinit.cpp

```diff
@@ -459,4 +435,12 @@ void usageTTL() {
     printf( "administrator (usually a few days).  With the --ttl option, you can\n" );
     printf( "specify how long this derived password will be valid, within the\n" );
     printf( "limits set by the administrator.\n" );
+    printf( "\n" );
+    printf( " -l  list the iRODS environment variables (only)\n" );
+    printf( " -v  verbose\n" );
+    printf( " -V  Very verbose\n" );
+    printf( "--ttl ttl  set the password Time To Live (specified in hours)\n" );
+    printf( "           Run 'iinit -h --ttl' for more\n" );
+    printf( " -h  this help\n" );
```

Comment:
Let's add an `Options:` header just above L439.

Consider aligning the option descriptions. Maybe `--ttl ttl`'s text is moved to the line below and aligned with the others.

Also, `--ttl ttl` -> `--ttl TTL`

Reply:
Sounds good

Reply:
Following commit addresses it. If it helps, here's what it looks like:
```
iinit -h
Creates a file containing your iRODS password in a scrambled form,
to be used automatically by the icommands.
Usage: iinit [-hvVl] [--ttl TTL]

When using regular iRODS passwords you can use --ttl (Time To Live)
to request a credential (a temporary password) that will be valid
for only the number of hours you specify (up to a limit set by the
administrator).  This is more secure, as this temporary password
(not your permanent one) will be stored in the obfuscated
credential file (.irodsA) for use by the other iCommands.

When using PAM, iinit always generates a temporary iRODS password
for use by the other iCommands, using a time-limit set by the
administrator (usually a few days).  With the --ttl option, you can
specify how long this derived password will be valid, within the
limits set by the administrator.

Options:
 -l  list the iRODS environment variables (only)
 -v  verbose
 -V  Very verbose
 --ttl TTL
     set the password Time To Live (specified in hours)
 -h  this help

iRODS Version 4.3.0                iinit
```
Not sure if that's what you had in mind

Reply:
Yes, that is correct. Just one last thing. Add a newline above the "Usage:" line and I think this PR will be ready.

Reply:
Done